miqa/core/tasks.py

```diff
@@ -49,6 +48,19 @@ def _download_from_s3(path: str, public: bool) -> bytes:
     return buf.getvalue()
 
 
+def datetime_string_valid(text):
```

Comment:
Can we just pull in the [dateparser](https://pypi.org/project/dateparser/) library and throw the string in, and if it can't figure it out, then we declare the date invalid? That seems easier than trying to enumerate all the valid date types, unless of course we want to restrict it to just these.

Reply:
Ooh, cool library. I haven't used it before. I'll try it out.